src/frameworks/web/components/atoms/SearchField/SearchField.tsx

```diff
@@ -0,0 +1,88 @@
+import styled from 'styled-components';
+import React from 'react';
+import theme from 'theme';
+import {
+  // eslint-disable-next-line no-unused-vars
+  _ISearchFieldContainer, ISearchField, ISearchFieldState,
+} from 'interfaces/frameworks/web/components/atoms/SearchField/ISearchField';
+
+const InputBox = styled.input`
+  width: 100%;
+  font-family: 'Muli', 'Noto Sans KR', sans-serif;
+  font-size: 16px;
+  font-weight: bold;
```

Comment:
```suggestion
  font-weight: normal;
```
제플린에서는 검색창 안 텍스트가 볼드가 아닌걸 이제야 봤네요 ㅠㅠ 상훈님이 볼드가 더 낫다고 생각하면 그대로 놔둬도 괜찮다고 생각합니다~